### ISSUE TITLE
Actuator coordinates

### ISSUE THREAD

**Issue by pytesnim** [AOtools/soapy#115]
Hello,
I would like to make a simulation for  deformable mirror ALPAO dmx37 with soapy. But I cant find out how to enter its hexagonal shape it seems soapy has only square grid of actuators.

**Comment by matthewtownson** [comment]
Yes, at the moment soapy only has square grids implemented for DMs, this is partially due to the simplicity of being able to implement them with square arrays.

If you have the time to implement a hexagonal grid then that would be a great feature to add, and we can try to support with any help you need. Unfortunately we don't have very much time at the moment, so I can't promise how much help we will be able to give.

**Comment by pytesnim** [comment]
Great, I will try. Thanks a lot!
